src/browser/storeImpl.ts

```diff
@@ -1,19 +1,23 @@
-import { defaultEngine, directoryHandleStoreKey } from "./contract";
+import { directoryHandleStoreKey } from "./contract";
 
-import {
-  configSchema,
-  ConfigType,
-  EngineId,
-  engineSettingSchema,
-} from "@/type/preload";
+import { BaseConfigManager, Metadata } from "@/shared/ConfigManager";
+import { ConfigType } from "@/type/preload";
 
-const dbName = `${import.meta.env.VITE_APP_NAME}-web`;
-const settingStoreKey = "electronStore";
-// FIXME: DBのschemaを変更したら、dbVersionを上げる
-// TODO: 気づけるようにしたい
+const dbName = `${process.env.APP_NAME}-web`;
+const settingStoreKey = "config";
 const dbVersion = 1;
```

Comment:
ちょっとメモ追加だけお願いしたく 🙇 
```suggestion
const dbVersion = 1; // FIXME: #1629 で不要になったはず
```

ここちょっと @yamachu さんと認識合わせたいかもです。

このPRで、 @sevenc-nanashi さんが作ってくれたConfigManager内でマイグレーションが行われる用になると思います。
ただこれはDBを開くときにマイグレーションする形ではなく、DB内にバージョン情報も書き込んでおいて、読み込んだ後にマイグレーションが必要であれば行って上書きする、という感じです。
マイグレーション処理の共通化のためにはこの形が良さそうに思うのですが、 @yamachu さん的にはどうでしょう 👀

Reply:
ですねーsharedで提供している機構を使ったconfigのみを使用するのであれば、これは完全固定値になって、openの時だけ見る謎の数字ぐらいになるのかなぁと。
https://github.com/VOICEVOX/voicevox/pull/1629/files?diff=split&w=0#diff-653b63cbc2a3f82b6039ddfa7ac72caf7cb9c879676febcc42753592c8a9bd12R42

Browserの提供しているIndexedDBのmigration機構を使わないので1以上の数字であって固定値であればまぁなんでもいいって感じですね。

例えばどうしてもブラウザ版では権限の制約で何かをキャッシュしなければならないみたいな状態だと、このStoreを使うことがあるのでどうしたもんかなぁとは思っています。

コメントを例えばconfigのmigrationには使用していない的なのを書くだけでも良かったりする…？
いやそうすると、じゃあなんやねんこの数字！ってなりますかね…

Reply:
なるほどです！！

> コメントを例えばconfigのmigrationには使用していない的なのを書くだけでも良かったりする…？

個人的には一旦これで良いのかなと思いました！
例えばこんな感じにしておいて、仕様が変わったときに再考かなと！

```suggestion
const dbVersion = 1; // 固定値。configのmigrationには使用していない。
```